src/index.html

```diff
@@ -1,66 +1,103 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset='UTF-8'>
     <title>Activity Tracker</title>
-    <link href="https://fonts.googleapis.com/css?family=Open%20Sans" rel='stylesheet'>
-    <link rel="stylesheet" type="text/css" href="./styles.css">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Open%20Sans' rel='stylesheet'>
```

Comment:
As previously discussed, all double quotes in the HTML were changed to single quotes. Functionality is unchanged.

Reply:
woohoo, thanks!